editors/code/package.json

```diff
@@ -243,16 +231,24 @@
           "default": true,
           "description": "Whether to show the 'Compile Code' icon in editor title menu."
         },
-        "sourcepawn.refreshServerPlugins": {
+        "sourcepawn.runServerCommands": {
           "type": "string",
           "enum": [
             "disabled",
             "afterCompile",
             "afterUpload"
           ],
-          "description": "Should the plugins list on the server be refreshed with `sm plugins refresh` after a successful action in VSCode.",
+          "description": "Should the specified commands be executed on the server after a successful action in VSCode (default: sm plugins refresh).",
```

Comment:
I don't think it's necessary to mention the default of another setting here. I understand that they are related, but this might confuse some people

Reply:
Oh I think I made a mistake, if I'm not wrong this clarification was supposed to go in the description of the new setting that actually contains the commands, but you're right, defaults are already there, so I should remove it altogether.